src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,24 @@
+package codeu.chat.common;
+
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+public final class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+  private Uuid parseResult;
```

Comment:
parseResult doesn't need to be a private member of the class, you only use it in the constructor.